modules/nf-core/bakta/baktadbdownload/meta.yml

```diff
@@ -24,7 +24,7 @@ output:
   - db:
       type: directory
       description: BAKTA database directory
-      pattern: "db/"
+      pattern: "db*/"
```

Comment:
to confirm: this shouldn't be `"db/*"` ?

Reply:
Thanks, but that is correct. Usually, even the asterisk is unnecessary, but it is fine to have here (as well as in the main.nf).